.projenrc.js

```diff
@@ -116,6 +116,7 @@ const depsForApiProject = [
   'i18next@^22.4.6',
   'i18next-fs-backend@^2.2.0',
   'fast-memoize@^2.5.1',
+  'lodash@^4.17.21',
```

Comment:
Should we move it to dev dependencies due to only using it in test code?

Reply:
Yes, moved it.